inst/eDISH_app/modules/renderSettings/renderSettings.R

```diff
@@ -210,8 +219,14 @@ renderSettings <- function(input, output, session, data, settings, status){
       }
     }
     
-    validateSettings(data(), settings_new, chart="eDish")
+    out <- list()
+    
```

Comment:
Interesting ... seems to work. Might be a bit slow if we had lots of charts. Maybe move the chart level validation to the chart tabs? But then I guess we couldn't show icons in the navbar ... hmm.

Reply:
Maybe we could calculate this automatically in `validateSettings` when running validation on a settings object covering multiple charts ...